mockup/patterns/relateditems/pattern.js

```diff
@@ -99,27 +99,30 @@ define([
       homeText: _t('home'),
       folderTypes: ['Folder'],
       selectableTypes: null, // null means everything is selectable, otherwise a list of strings to match types that are selectable
-      attributes: ['UID', 'Title', 'portal_type', 'path', 'getIcon'],
+      attributes: ['UID', 'Title', 'portal_type', 'path','getURL', 'getIcon','contenttype_class','is_folderish','review_state'],
```

Comment:
I would suggest adding a function here: https://github.com/plone/mockup/blob/master/mockup/js/utils.js to format portal_type class and that we try to use that everywhere in the js when we want font-icons to work. Does that help?